indexing-handlers/src/main/java/no/unit/nva/indexing/handlers/IndexResourceHandler.java

```diff
@@ -87,4 +89,9 @@ private IndexDocument fetchFileFromS3Bucket(UnixPath resourceRelativePath) {
         var resource = resourcesS3Driver.getFile(resourceRelativePath);
         return IndexDocument.fromJsonString(resource);
     }
+
+    private IndexDocument expandResource(IndexDocument indexDocument) {
+        ResourceExpansion.expand(indexDocument.resource());
+        return indexDocument;
+    }
```

Comment:
void method

Reply:
then we end up with only 
`ResourceExpansion.expand(indexDocument.resource());`